Module/Rule.SqlScriptQuery/Convert/Methods.ps1

```diff
@@ -484,107 +483,686 @@ function Get-AuditSetScript
 
     return $sqlScript
 }
+
+<#
+    .SYNOPSIS Get-AuditEvents
+        Returns a string of the audit events found in CheckContent
+
+    .DESCRIPTION
+        This function returns the audit events found in CheckContent as a comma-delimited string, suitable for insertion into a SQL statement.
+
+    .PARAMETER FixText
+        String that was obtained from the 'CheckContent' element of the base STIG Rule
+
+    .PARAMETER CheckContent
+        The STIG content that contains possible audit events
+#>
+function Get-AuditEvents
+{
+    [CmdletBinding()]
+    [OutputType([string])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [AllowEmptyString()]
+        [string[]]
+        $CheckContent
+    )
+
+    $collection = @()
+    $pattern = '([A-Z_]+)_GROUP(?!\x27|\x22)'
+    foreach ($line in $CheckContent)
+    {
+        $auditEvents = $line | Select-String -Pattern $pattern -AllMatches
+        foreach ($auditEvent in $auditEvents.Matches)
+        {
+            $collection += $auditEvent
+        }
+    }
+    # Return an array of found SQL audit events
+    return $collection
+}
+#endregion Audit Functions
+
+#region PlainSQL Functions
+<#
+    .SYNOPSIS
+        Returns a plain SQL query from $CheckContent
+
+    .DESCRIPTION
+        The SqlScriptResource uses a script resource format with GetScript, TestScript and SetScript.
+        The SQL STIG contains queries that will be placed in each of those blocks.
+        This function returns the query that will be used in the GetScript block
+
+    .PARAMETER CheckContent
+        This is the 'CheckContent' derived from the STIG raw string and holds the query that will be returned
+#>
+function Get-PlainSQLGetScript
+{
+    [CmdletBinding()]
+    [OutputType([string])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [AllowEmptyString()]
+        [string[]]
+        $CheckContent
+    )
+
+    $return = Get-SQLQuery -CheckContent $CheckContent
+
+    return $return
+}
+
+<#
+    .SYNOPSIS
+        Returns a T-SQL query
+
+    .DESCRIPTION
+        The SqlScriptResource uses a script resource format with GetScript, TestScript and SetScript.
+        The SQL STIG contains queries that will be placed in each of those blocks.
+        This function returns the query that will be used in the TestScript block
+
+    .PARAMETER CheckContent
+        This is the 'CheckContent' derived from the STIG raw string and holds the query that will be returned
+#>
+function Get-PlainSQLTestScript
+{
+    [CmdletBinding()]
+    [OutputType([string])]
+    param
+    (
+        [Parameter(Mandatory = $true)]
+        [AllowEmptyString()]
+        [string[]]
+        $CheckContent
+    )
+
+    $return = Get-SQLQuery -CheckContent $CheckContent
+
+    return $return
+}
+
```

Comment:
Why do we need both Get-PlainSQLGetScript and Get-PlainSQLTestScript?  Both appear to perform the same function.